src/App.tsx

```diff
@@ -1,17 +1,108 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+
+import { TodoLis } from './components/TodoList/TodoList';
+
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const listTodoWithUser = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [listTodo, setListTodo] = useState([...listTodoWithUser]);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titelIsEmpty, setTitelISEmpty] = useState('');
+  const [idIsEmpty, setIdIsEmpty] = useState('');
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitelISEmpty('');
+  };
+
+  const handleUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setIdIsEmpty('');
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    if (!title.length || !userId) {
+      if (!title.length) {
+        setTitelISEmpty('Please enter the title');
+      }
+
+      if (!userId) {
+        setIdIsEmpty('Please choose a user');
+      }
+    } else {
```

Comment:
try to avoid using else
```suggestion
      if (!title.length) {
        setTitelISEmpty('Please enter the title');
        
        return;
      }

      if (!userId) {
        setIdIsEmpty('Please choose a user');
        
        return;
      }
```